lib/modules/manager/gomod/artifacts.ts

```diff
@@ -133,7 +133,9 @@ export async function updateArtifacts({
 
   const vendorDir = upath.join(goModDir, 'vendor/');
   const vendorModulesFileName = upath.join(vendorDir, 'modules.txt');
-  const useVendor = (await readLocalFile(vendorModulesFileName)) !== null;
+  const useVendor =
+    (await readLocalFile(vendorModulesFileName)) !== null &&
+    !config.postUpdateOptions?.includes('gomodVendorSkip');
```

Comment:
invert condition

Reply:
Do you mean as in first checking if the option is not set and then checking the file, or as in flipping the meaning of the variable to something like `skipVendor`?

Reply:
yes, first checking the option. that should be faster and we don't need to read the file if we skip anyways

Reply:
Right, I'll fix it.